script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_5B.txt

```diff
@@ -1,31 +1,31 @@
 <Page1>
-It was just like a human missile.
+She was just like a human missile.
 Arcueid descended from a height of 80 meters above Vlov.
 <Page2>
 No, it was more like she flew.
 While in the air, she took aim and angled herself before accelerating like a missile.
 <Page3>
-Sand and debris billow up into the air.
-The wind whipped up a cloud of dust that sweeps the area.
-The river of flame that had swept across the downtown area was eradicated by that single blow.
+Earth and debris billow up into the air.
+The wind whipped up a cloud of dust that scoured the area.
+The river of flame that had been flowing over the downtown area was halted by that single blow.
 <Page4>
-The policemen that surround Vlov in the distance are blown away, incapacitating most of them. They are divided into two groups: those who became unconscious after suffering a head injury and those who lose their will to fight.
+The policemen that had been encircling Vlov are blown away, and most of them are incapacitated in the process. Half of them have been knocked unconscious by blows to the head, and the rest have simply lost their will to fight.
```

Comment:
incapacitating most of them in the process.
(I feel like the 'and' slows the flow of the sentence down and is already used in the next one.)

Reply:
Hmm, that doesn't quite read right to me:
> The policemen that had been encircling Vlov are blown away, incapacitating most of them in the process.

It feels like it's missing a reference to what is _doing_ the incapacitating. Maybe
> The blast blows away the policemen that had been encircling Vlov, incapacitating most of them in the process.

?

Reply:
I assumed that being blown away was what incapacitated them.

Your new sentence with the blast works though.

Reply:
It is, but for some reason that formulation of the sentence feels like it skips something to me /shrug
Will use the second one.